src/test/java/uk/ac/ebi/subs/fileupload/eventhandlers/PostCreateEventTest.java

```diff
@@ -23,9 +23,10 @@
 public class PostCreateEventTest {
 
     private TUSFileInfo tusFileInfo;
-    private static final String JWT_TOKEN = "xxxxx.yyyyy.zzzz";
+    private static final String JWT_TOKEN = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE1MTYzNjk4NTEsImV4cCI6MTU0NzkwNTg1MSwiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIm5hbWUiOiJLYXJlbCIsIkVtYWlsIjoia2FyZWxAZXhhbXBsZS5jb20iLCJEb21haW5zIjpbInRlYW1fYSIsInRlYW1fYiJdfQ.uGvNNVZZjb3CNc0zX5zj_QPz2pOAGZ7HQZbFeCU7a7g";
```

Comment:
Can you talk me through why we need this token here?

Reply:
You are right, we don't need a real token, specially because it is expiring and I'm mocking my validation service, so I don't really need it, just as a dummy one.